controllers/bootstrap/init.go

```diff
@@ -212,7 +220,15 @@ func (b *Bootstrap) InitResources(manualManagement bool) error {
 		return err
 	}
 
-	if err := b.createOrUpdateResources(annotations, OdlmCrResources); err != nil {
+	if err := b.createOrUpdateResources(annotations, RegistryCrResources); err != nil {
```

Comment:
Do we need to remain the second parameter as the string slice in this case?
or we could use func `createOrUpdateResource` if it is only the string.

Reply:
Nice catch, updated